python/lsst/meas/algorithms/astrometrySourceSelector.py

```diff
@@ -0,0 +1,121 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+from lsst.afw import table
```

Comment:
Apparently we never settled on a namespace standard in RFC-34: https://jira.lsstcorp.org/browse/RFC-34. I guess I'll go with the LSST "standard" for now.